### DISCUSSION TITLE
/zh-cn/blog/posts/svg_1/

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [Reverier-Xu/discussions#119] [Announcements]
https://blog.woooo.tech/posts/svg_1/?code=33880ab340af35772bcc

**Comment by arttnba3** [comment]
RX是SVG~~批~~爱好者，👴是RX~~批~~爱好者，所以👴也是SVG~~批~~爱好者（（

**Comment by Reverier-Xu** [comment]
> @arttnba3
> RX是SVG~~批~~爱好者，👴是RX~~批~~爱好者，所以👴也是SVG~~批~~爱好者（（

你是个**

**Comment by arttnba3** [comment]
> @Reverier-Xu
> > @arttnba3
> > RX是SVG~~批~~爱好者，👴是RX~~批~~爱好者，所以👴也是SVG~~批~~爱好者（（
> 
> 你是个**

# 😭😭😭
